AUTOTEST/runtests-sstruct-mixed

```diff
@@ -0,0 +1,4 @@
+TEST_sstruct/[^n]*.sh
+TEST_sstruct/neighborpart.sh
+TEST_sstruct/neumann.sh
+
```

Comment:
I was looking for a slightly cleaner way than this to exclude file names that start with `non`. No luck so far, not as easy as I thought. But this looks fine to me.

Reply:
Agreed.  Without using things like awk or sed, I don't know how to do it.  The wildcard stuff in linux is somewhat limited in what can be expressed.  If I come up with a clever idea, I'll let you know.  I am happy (I think) with the use of the individual files to specify the runtest tests.  The format of what goes in them could be improved.